Packs/FortiGate/TestPlaybooks/playbook-FortiGate_Test.yml

```diff
@@ -0,0 +1,5677 @@
+id: FortiGate
```

Comment:
If this is a new test-PB reduce to version -1

Reply:
reduced.